pkg/rollup/config.go

```diff
@@ -0,0 +1,82 @@
+package rollup
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/jackc/pgx/v4"
+
+	"github.com/timescale/promscale/pkg/internal/day"
+	"github.com/timescale/promscale/pkg/log"
+)
+
+type Config struct {
+	Enable               *bool `yaml:"enabled,omitempty"`
+	UseDefaultResolution *bool `yaml:"use_default_resolution"`
+	Resolution           `yaml:"resolutions,omitempty"`
+}
+
+type Info struct {
+	Resolution day.Duration `yaml:"resolution"`
+	Retention  day.Duration `yaml:"retention"`
+	Delete     bool         `yaml:"delete"`
+}
+
+type (
+	Label      string
+	Resolution map[Label]Info
+)
+
+var (
+	setDefaultDownsampleStateSQL = "SELECT prom_api.set_automatic_downsample($1)"
+	defaultDownsampleState       = false
+	useDefaultResolution         = true
```

Comment:
Should these 3 be const instead of var?

Reply:
We can for `setDefaultDownsampleStateSQL` but not for the rest two, since we use the address of the rest two.